### PR TITLE
splash:0.3.0

### PR DESCRIPTION
**Pull request by kaarmu**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: This library provides different color palettes with human-readable names in Typst dictionaries.

### CODE REVIEW

**Comment by reknih** [comment]
Hey, thanks for your package submission! Could you add a bit of "pazazz" to the package name? As-is, it's merely descriptive, which could deceive users into assuming this is an officially endorsed package.

**Comment by kaarmu** [comment]
Thank you for the response! Sure thing.

**Comment by kaarmu** [comment]
I hope this name has enough pazazz :smiley:

**Comment by reknih** [comment]
It sure has!

File: packages/preview/splash-0.3.0/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+A library of color palettes for Typst.
+
+This library provides different color palettes with human-readable names in
+Typst dictionaries. Currently there are just a few different palettes to choose
+from. Any contributions or suggestions are welcome!
+
+## Usage
+
+```typst
+#import "palettes.typ": xcolor
```
Comment (laurmaedje) [inline packages/preview/splash-0.3.0/README.md:12]: This import should show how to import with the package manager

File: packages/preview/splash-0.3.0/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# typst-palettes
```
Comment (laurmaedje) [inline packages/preview/splash-0.3.0/README.md:1]: This should be `splash`

File: packages/preview/splash-0.3.0/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+A library of color palettes for Typst.
+
+This library provides different color palettes with human-readable names in
+Typst dictionaries. Currently there are just a few different palettes to choose
+from. Any contributions or suggestions are welcome!
```
Comment (laurmaedje) [inline packages/preview/splash-0.3.0/README.md:7]: Ideally the README would also state that you should look into the Typst files for the palette and color names or maybe list the available palettes here.

Comment (kaarmu) [reply packages/preview/splash-0.3.0/README.md:7]: There is a generated document with all the colors but I don't want to include it in the distributed files due to the unnecessary added size. I hope this is sufficient.

Comment (laurmaedje) [reply packages/preview/splash-0.3.0/README.md:7]: Ah that's great. Then, I would propose linking "generated documentation" directly to that PDF.

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**